runtime/server/connector_service.go

```diff
@@ -141,10 +143,16 @@ func (s *Server) OLAPListTables(ctx context.Context, req *runtimev1.OLAPListTabl
 	}
 	defer release()
 
-	tables, err := olap.InformationSchema().All(ctx, req.SearchPattern)
+	i := olap.InformationSchema()
+	tables, err := i.All(ctx, req.SearchPattern)
 	if err != nil {
 		return nil, err
 	}
+	err = i.LoadPhysicalSize(ctx, tables)
+	if err != nil {
+		// the size queries can fail due to permission erros so we log them and continue
+		s.logger.Warn("failed to populate table sizes", zap.Error(err))
+	}
```

Comment:
I'm a bit worried we'll see a lot of these, especially if the cloud UI starts fetching table sizes. I'd be up for just discarding the error (or turning it into an internal debug log), and just letting it return `-1`.

Also, if you intend to keep a log, should we add `observability.ZapCtx(ctx)`?